modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,299 @@
+
+** In the trigger specifications, you can configure interceptors using a new API that includes a `ref` field to refer to a cluster interceptor. In addition, you can use the `params` field to add parameters that will be passed on to the interceptors for processing.
+
+** The bundled interceptors CEL, GitHub, GitLab, and BitBucket, have been migrated. They are implemented using the new `ClusterInterceptor` custom resource definition.
+
+** Core interceptors are migrated to the new format, and any new triggers created using the old syntax will automatically switch to the new `ref` or `params` based syntax.
+
+* To disable prefixing the name of the task or step while displaying logs, you can use the `--prefix` option for `log` commands.
+
+* To display the version of a specific component, use the new `--component` flag in the `tkn version` command.
+
+* The `tkn hub check-upgrade` command is added, and other commands are revised to be based on the pipeline version. In addition, catalog names are displayed in the `search` command output.
+
+* Support for optional workspaces are added to the `start` command.
+
+* The plugins will be looked up in the current path, if they are not present in the `plugins` directory.
+
+* The `tkn [task | clustertask | pipeline] start` command will start interactively and ask for the `params` value, if default is not specified or passed through flag.
+
+* The `version` field is added in the `tkn task describe` command.
+
+* The option to automatically select `TriggerTemplate`, or `TriggerBinding`, or `ClusterTriggerBinding`, or `Eventlistener`, has been added in the `describe` command, if only one is present.
+
+* In the `tkn pr describe` command, a section for skipped tasks is added.
+
+* Support for the `tkn clustertask logs` has been added.
+
+* The YAML merge and variable from `config.yaml` has been removed. In addition, the `release.yaml` file can now be more easily consumed by tools like `kustomize` and `ytt`.
+
+* The support for resource names to contain the dot character (".") has been added.
+
+* The `hostAliases` in the `PodTemplate` specification has been added to the pod-level override of hostname resolution. It ha been achieved by modifying the `/etc/hosts`.
+
+* Steps in a task no longer receive a default `HOME` environment variable of `/tekton/home`, nor a default working directory of `/workspace`, as it is disabled.
+
+* A variable `$(tasks.status)` has been introduced to access the aggregate execution status of tasks.
+
+* An entry-point binary build for Windows has been added.
+
+
+[id="deprecated-features-1-5_{context}"]
+== Deprecated features
+
+* In the `when` expressions, support for fields written is PascalCase is removed. The `when` expressions only support fields written in lowercase.
++
+[NOTE]
+====
+If you had applied a pipeline with `WhenExpressions` in v0.16, you have to reapply it.
```

Comment:
Do we need to specify the pipelines version (`v0.16`) or the operator version (`1.2`) here?

Reply:
either way we have to mention what version it signifies.
v0.16 (pipelnes) or v1.2 (operator)

i prefer v1.2.x (operator)